pulser-pasqal/pulser_pasqal/pasqal_cloud.py

```diff
@@ -100,7 +100,12 @@ def __init__(
         )
 
     def submit(
-        self, sequence: Sequence, wait: bool = False, **kwargs: Any
+        self,
+        sequence: Sequence,
+        wait: bool = False,
+        complete: bool = True,
```

Comment:
You are calling this parameter `open` elsewhere

Reply:
Ah yeah I'll clean this up. We're discussing to swap from complete -> open elsewhere offline my mind slipped on this

Reply:
So, we'll go with `open` in the end, right?

Reply:
Ok, but in that case you have to change it in the `__enter__()` because there it's `open` atm. Also, I think `complete` warrants a small comment because to me the meaning is not obvious from the name (especially if I was unfamiliar with the Cloud-SDK)

Reply:
Yeah, I'm aware it's not clear. Poor naming conventions is popular cloud side, cc rebatch

I'll add a comment. We have changes down the line to rename this so once the SDK has that change we can make an update here in the future

Reply:
We have an open MR to rename `complete` to `open` in the SDK:  https://github.com/pasqal-io/pasqal-cloud/pull/131

So we can name this `open` in pulser-pasqal right away

Reply:
I was sure I left a comment on this yesterday but I can't find it again, apologies if this is a duplicate.

There is an open [MR](https://github.com/pasqal-io/pasqal-cloud/pull/131) to rename `complete` to `open` in the SDK so it should be named `open` right away in pulser pasqal

Reply:
Yeah, I just replied to your comment? We'll need to wait until the MR goes in and I can bump the SDK version otherwise it's only going to be referred to as `open` on the pulser-pasqal side